car.csv

```diff
@@ -0,0 +1,2 @@
+this, is, a, car, file
```

Comment:
is this file needed to make the simulation work? If not lets leave out of the code base